pkg/nsx/util/utils.go

```diff
@@ -311,6 +311,19 @@ func DumpHttpRequest(request *http.Request) {
 	log.V(2).Info("http request", "url", request.URL, "body", string(body), "head", request.Header)
 }
 
+// NSXApiError processes an error and returns a formatted NSX API error message if applicable.
+// If the processed API error is nil, return the original error
+func NSXApiError(err error) error {
+	if err == nil {
+		return err
+	}
+	apierror, _ := DumpAPIError(err)
+	if apierror == nil {
+		return err
+	}
+	return fmt.Errorf("nsx error: %s", *apierror.ErrorMessage)
+}
```

Comment:
could you print all the apierror instead of just ErrorMessage?

Reply:
ok